client/assets/stylesheets/shared/functions/_z-index.scss

```diff
@@ -171,6 +171,7 @@ $z-layers: (
 		'.dialog__backdrop': 100200,
 		'.wplink__dialog.dialog.card': 100200,
 		'.web-preview': 100200,
+		'.plan-storage__tooltip': 100300,
```

Comment:
As the reakit tooltip sits outside the page flow at the end of the page, we need a high enough z-index higher than the media library modal at 100200.